### PR TITLE
Patch 7

### PR DESCRIPTION
**Pull request by dariomws306**
I think we should remove Complete-CsCceApplianceRegistration and Complete-CsCceApplianceUnregistration because they doesn't exist. I have verified that with SfBO Powershell and PlatyPS.

### CODE REVIEW

**Comment by kenwith** [comment]
@tomkau - next batch for your review.

File: skype/skype-ps/skype/Get-CsCallerIdPolicy.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ schema: 2.0.0
 # Get-CsCallerIdPolicy
 
 ## SYNOPSIS
-Provide the topic introduction here.
+This cmdlet has been deprecated please use Get-CsCallingLineIdentity to create new caller id policy.
```
Comment (tomkau) [inline skype/skype-ps/skype/Get-CsCallerIdPolicy.md]: Should not say "to create new".. should say "to get"

Comment (tomkau) [reply skype/skype-ps/skype/Get-CsCallerIdPolicy.md]: I see this same comment repeated for the other CsCallerId cmdlets too.   Let's use a more generic comment:

"The CsCallerIdPolicy cmdlets have been deprecated.  Please use the CsCallingLineIdentity cmdlets to manage caller ID policies for your organization.

Comment (dariomws306) [reply skype/skype-ps/skype/Get-CsCallerIdPolicy.md]: done, thanks!

**Comment by dariomws306** [comment]
Thanks! Please check Complete-CsCceApplianceRegistration.md, I think it should be removed, I cannot see it in SfB Online Powershell neither in PlatyPS. 

Also I couldn't test  Set-CsCceApplianceStatus and Set-CsCceApplianceDeploymentStatus because to deploy Cloud Connector Edition I need a Hyper-V host with at least 32GB of RAM.

Thanks.

**Comment by tomkau** [comment]
Complete-CsCceApplianceRegistration (and Unregistration) is still present in the code, so let's not remove the help file.   It is an on-prem cmdlet, so will not be present in the online connector module.

**Comment by dariomws306** [comment]
Ok, I looked for it there because it said "applicable Skype for Business Online". I am trying now in a SfB OnPremises Front End with the latest update and I cannot see it any Complete-CsCce* cmdlet. Maybe in CCE?

Thanks.

**Comment by tomkau** [comment]
Yes, this is specific to CCE.  It does require an online subscriptions, but it's not part of the online PS connector.

**Comment by dariomws306** [comment]
Also couldn't find it here:
https://docs.microsoft.com/en-us/skypeforbusiness/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/cloud-connector-cmdlet-reference

I will try to build a cce lab...

**Comment by kenwith** [comment]
@tomkau - can I go ahead and merge this commit? We can circle back to CCE after we get through the other Online cmdlets. What do you think?

**Comment by kenwith** [comment]
@tomkau - @dariomws306 is already starting on another batch. Can we go ahead and merge this one?

**Comment by tomkau** [comment]
Yes, OK to merge.

**Pull request merged**